src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -43,25 +43,38 @@
   
   <PropertyGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
     <DefineConstants>FUNCTIONS_V1</DefineConstants>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
   </PropertyGroup>
 
+  <ItemGroup>
+    <!--Don't include Correlation classes in Function V1.-->
+    <Compile Include="**/*.cs" Exclude="Correlation/*.cs;**/obj/**/*.cs" Condition="'$(TargetFramework)' != 'netstandard2.0'" />
+  </ItemGroup>
+
   <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="2.3.0" />
 
     <!--Don't increase below versions without significantly testing on Functions V1. Increasing these versions increments
     some dependencies that have binding redirects in Functions V1.-->
     <PackageReference Include="Microsoft.Extensions.Options" Version="1.0.2" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="1.0.2" />
-    <PackageReference Include="Microsoft.AspNetCore.Routing" Version="1.0.3" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.WebApiCompatShim" Version="1.0.3" />
+    <PackageReference Include="Microsoft.AspNetCore.Routing" Version="1.0.5" /> <!-- used to be 1.0.3 -->
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.WebApiCompatShim" Version="1.0.5" /> <!-- used to be 1.0.3 -->
+    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.2.3" />
```

Comment:
This seems incorrect now.

Reply:
@TsuyoshiUshio do you know why this changed?

Reply:
We may be able to look at the original PR that created the correlation branch, as that had several iterations of PR review and it may have came up there.

Reply:
It should be match with the DurableTask.Core version that includes Correlation feature.

Reply:
This has been simplified in the my last few commits